data/flags/man_made/flagpole.json

```diff
@@ -2902,6 +2902,21 @@
         "subject:wikidata": "Q1030"
       }
     },
+    {
+      "displayName": "Navajo Nation",
+      "id": "navajonation-e5dc93",
+      "locationSet": {"include": ["001"]},
+      "matchNames": ["navajo"],
+      "tags": {
+        "flag:colour": "orange",
```

Comment:
I think the field is supposed to be buff or tan. Orange is a ways off, and there doesn’t seem to be a CSS color keyword for buff.